source/_docs/automation/templating.markdown

```diff
@@ -9,6 +9,15 @@ Automations support [templating](/docs/configuration/templating/) in the same wa
 
 The following tables show the available trigger data per platform.
 
+### All
```

Comment:
If we make an `All` chapter, we should move some others as well.

For example, `trigger.platform` is also available in all trigger types.

Reply:
@frenck Since `trigger.platform` has a distinct, hardcoded value that differs between triggers, what would you suggest here? I could add a high-level, generic entry for `trigger.platform` under “All”? It makes sense to leave the current individual triggers with their existing entries so the specific hardcoded values are documented clearly.

Reply:
Hmm good point, lets leave it like this is 👍